#!/usr/bin/python3


class PowerDialerError(Exception):
    """An error related to the PowerDialer"""
